pkg/ingestor/parser/cyclonedx/parser_cyclonedx.go

```diff
@@ -78,8 +78,22 @@ func NewCycloneDXParser() common.DocumentParser {
 	}
 }
 
+// initializeCDXParser clears out all values for the next iteration
+func (c *cyclonedxParser) initializeCDXParser() {
```

Comment:
`initializeParser` - was added to cleanup the parser when the ingestor is run as a service. Otherwise, it resulted in wrong ingestion due to old data not being cleared with each ingestion.